docs/samples/browser-plugin-meetings/app.js

```diff
@@ -409,23 +414,23 @@ function verifyPassword() {
     meeting
       .verifyPassword(joinOptions.pin, joinOptions.captcha)
       .then((res) => {
-        if (res.failureReason === 'NONE') {
+        /* we have a bug in the SDK code: for case if wbxappapi requires only captcha and no password */
+        if (res.isPasswordValid) {
           passwordCaptchaStatusElm.innerText = 'Password is verified';
           passwordCaptchaStatusElm.style.backgroundColor = '#49e849';
           verifyPasswordElm.disabled = true;
           document.getElementById('btn-join').disabled = false;
           document.getElementById('btn-join-media').disabled = false;
-          // joinMeetingNow();
-        } else if (res.failureReason === "WRONG_PASSWORD") {
-          passwordCaptchaStatusElm.innerText = 'Password is required';
-          passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
-        } else if (res.failureReason === 'WRONG_CAPTCHA') {
+        } else if (!res.isPasswordValid && res.requiredCaptcha) {
```

Comment:
This `!res.isPasswordValid` looks like unwanted check. `res.isPasswordValid` will be false or undefined then only it can come to this line and ! of false or undefined will be true always.

Reply:
Yes, Correct @meta-dipanshu-sharma. I ended up adding this. Let me correct it and commit that changes.

Reply:
I have incorporated the comments, Please check once @meta-dipanshu-sharma